amundsen_application/static/js/ducks/search/tests/index.spec.ts

```diff
@@ -0,0 +1,211 @@
+import { expectSaga, testSaga } from 'redux-saga-test-plan';
+import * as matchers from 'redux-saga-test-plan/matchers';
+import { throwError } from 'redux-saga-test-plan/providers';
+
+import { ResourceType } from 'interfaces';
+
+import globalState from 'fixtures/globalState';
+
+import { searchResource as srchResource } from '../api/v0';
```

Comment:
i plan on renaming so that we don't have these conflicts, but doing that outside of PRs -- didn't want to cause any further confusion in the large diffs.